assets/queries/ansible/azure/ssl_connection_is_enabled/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ansible_ssl_connection_is_enabled",
+  "queryName": "SSL Connection Is Enabled",
+  "severity": "HIGH",
+  "category": "Network Security",
+  "descriptionText": "Make sure that for MySQL Database Server, 'Enforce SSL connection' is enabled",
+  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/mysql_server"
```

Comment:
Change to ansible relevant documentation

Reply:
Done :)